frontend/src/components/regions/DueTimeFormModal/index.jsx

```diff
@@ -11,7 +11,10 @@ function DueTimeFormModalUnconnected({
   card,
   handleClose,
   fetchAgileCard,
+  setDueTime,
+  CARD_SET_DUE_TIME,
```

Comment:
I think this will be undefined: CARD_SET_DUE_TIME


where is it coming from? Did you mean to update mapStateToProps?

Reply:
Resolved. Added field to mapStateToProps along with check for getting the latestApiCallStatus for it as well